scenariolib/visit.go

```diff
@@ -189,15 +189,17 @@ func (v *Visit) sendViewEvent(rank int, contentType string, pageViewField string
 
 	event.PageURI = v.LastResponse.Results[rank].ClickURI
```

Comment:
PageURI and PageTitle do not exist with this name in the viewEvent in Usage Analytics. They are named location and title. It would make sense to use the same field names. Also, you could add the contentIdValue that seems to be missing

Reply:
Ok, thanks will change the names.

The contentIdValue will be pulled from the result[pageViewField]. 

`The view event sent will contain {contentIdKey: "@pageViewField", contentIdValue: result['pageViewField']}`

See the comment on the ViewEvent type in event_pageview.go.